_pages/about-us/teams/products-and-platforms.md

```diff
@@ -15,13 +17,13 @@ Products and Platforms is one of the 5 business units within 18F. We strive to i
 
 * 18F's work on the [eRegs platform](https://eregs.github.io/) with various partners.
 
-* With partners in TTS, the [Draft US Web Design Standards](https://standards.usa.gov/): open-source UI components and visual styles to create beautiful, consistent experiences across federal government websites.
```

Comment:
Sounds good! And if most readers are already familiar with the term, we may not need to define it in text — I just wasn't sure of readers' familiarity with TTS.